modules/assignment/src/main/java/com/intuit/wasabi/assignment/AssignmentsModule.java

```diff
@@ -100,6 +111,42 @@ protected void configure() {
         LOGGER.debug("installed module: {}", AssignmentsModule.class.getSimpleName());
     }
 
+    private void bindMetadataCache(final Properties properties) {
+        Boolean metadataCacheEnabled = Boolean.parseBoolean(getProperty("metadata.cache.enabled", properties, "true"));
+        bind(Boolean.class).annotatedWith(named("AssignmentsMetadataCacheEnabled")).toInstance(metadataCacheEnabled);
+
+        if(metadataCacheEnabled) {
+            //This is a cache refresh interval, at this frequency cache will be refreshed.
+            Integer metadataCacheRefreshInterval = Integer.parseInt(getProperty("metadata.cache.refresh.interval", properties, "5"));
```

Comment:
It might be helpful if we add units to the property name - eg:  ...interval.millis or inteval.sec.